compiler-core/src/erlang/tests/snapshots/gleam_core__erlang__tests__guards__field_access.snap

```diff
@@ -0,0 +1,24 @@
+---
+source: compiler-core/src/erlang/tests/case.rs
+assertion_line: 23
+expression: "\npub fn myfun(mt) {\n  case mt {\n    1 | _ ->\n      1\n      |> Ok\n  }\n  1\n  |> Ok\n}\n"
+---
+-module(my@mod).
+-compile([no_auto_import, nowarn_unused_vars]).
+
+-export([main/0]).
+-export_type([person/0]).
+
+-type person() :: {person, binary(), binary(), integer()}.
+
+-spec main() -> binary().
+main() ->
+    Given_name = <<"jill"/utf8>>,
+    Person = {person, <<"credo"/utf8>>, <<"jack"/utf8>>, 32},
+    case Given_name of
+        Name when Name =:= erlang:element(2, Person) ->
```

Comment:
The first element in the tuple is the the tag, so this would need to get the 3rd element rather than the 2nd (Erlang tuples are 1 indexed)

Reply:
I think I forgot to update the snapshot, it seems to be fixed now!